packages/mobile/src/btoa.ts

```diff
@@ -9,3 +10,6 @@ if (typeof global.self === 'undefined') {
   global.self = global
 }
 global.btoa = require('Base64').btoa
+// Without this, one will see a confusing error
+// similar to https://imgur.com/a/7rnLIh5
+global.URL = require('whatwg-url').URL
```

Comment:
Not strongly opposed to adding this here but please rename the file. Maybe `missingGlobals.ts`

Reply:
Done.